examples/06-mnist.rs

```diff
@@ -36,7 +29,12 @@ struct MnistDataset {
 
 impl MnistDataset {
     fn train(path: &str) -> Self {
-        let mnist: Mnist = MnistBuilder::new().base_path(path).finalize();
+        let mnist: Mnist = MnistBuilder::new()
+            .base_url("http://yann.lecun.com/exdb/mnist/")
+            // .base_url("https://cmoran.xyz/data/mnist/standard/")
```

Comment:
Yeah let's just keep the yann lecun ones since that is pretty standard. Great addition though!